App.js

```diff
@@ -1,26 +1,70 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/native'
+import { ActivityIndicator } from 'react-native'
+import { Movie } from './components/Movie'
 
-const App = () => {
-  return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
+const TOP_MOVIE_ID = 656516
 
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+  background-color: black;
+  align-items:center;
```

Comment:
To make sure that the `ActivityIndicator` shows, add some padding to the top of this container
` padding-top: 60px;` for example.